views/endpoints.ejs

```diff
@@ -2,29 +2,52 @@
     <div class="row">
         <% if(!endpoints.length) { %>
         <h1>No endpoints found</h1>
         <% } else { %>
         <% endpoints.forEach(function(endpoint) { %>
         <div class="col-md-3 g-3">
-            <div class="card fixed-sized-card">
+            <div class="card fixed-size-card">
                 <div class="card-body">
                     <% if(endpoint.platform=="rhel") { %>
                     <img src="/img/rhel_logo.png" class="card-img-top platform-img">
-                    <% } else {; %>
-                    <img src="/img/ubuntu.png" class="card-img-top platform-img">
+                    <% } else if(endpoint.platform=="kali") { %>
+                    <img src="/img/kali_logo.png" class="card-img-top platform-img">
+                    <% } else if(endpoint.platform=="ubuntu") { %>
+                    <img src="/img/ubuntu_logo.png" class="card-img-top platform-img">
+                    <% } else if(endpoint.platform=="debian") { %>
+                    <img src="/img/debian_logo.png" class="card-img-top platform-img">
+                    <% } else if(endpoint.platform=="centos") { %>
+                    <img src="/img/centos_logo.png" class="card-img-top platform-img">
+                    <% } else if(endpoint.platform=="macos") { %>
+                    <img src="/img/macos_logo.png" class="card-img-top platform-img">
+                    <% } else if(endpoint.platform=="windows") { %>
+                    <img src="/img/windows_logo.png" class="card-img-top platform-img">
+                    <% } else { %>
+                    <img src="/img/generic_logo.png" class="card-img-top platform-img">
                     <% } %>
-                    <h5 class="card-title">
+                    <h4 class="card-title">
                         <%= endpoint.hostname %>
-                    </h5>
+                    </h4>
+                    <p class="card-text">
+                        <strong>IP:</strong> <%= endpoint.primary_ip %>
+                    </p>
+                    <p class="card-text">
+                        <strong>MAC:</strong> <%= endpoint.primary_mac %>
+                    </p>
                     <p class="card-text">
-                        <%= endpoint.primary_ip %>
+                        <strong>OS:</strong> <%= endpoint.os_version %>
                     </p>
                     <p class="card-text">
-                        <%= endpoint.primary_mac %>
+                        <strong>Uptime:</strong> <%= (endpoint.uptime / (1000 * 60 * 60 * 24)).toFixed(2) %> days
+                    </p>
+                    <p class="card-text">
+                        <strong>Last Seen:</strong> <%= new Date(endpoint.seen_time).toLocaleString() %>
+                    </p>
+                    <p class="card-text">
+                        <strong>Last Scan:</strong> <%= endpoint.last_execution ? new Date(endpoint.last_execution).toLocaleString() : 'N/A' %>
```

Comment:
@maziland here should be replaced by data from the Scripts API
Do you want to check what I suggested on https://biosquery.atlassian.net/browse/BQU-16?